master/internal/api_model.go

```diff
@@ -189,12 +231,29 @@ func (a *apiServer) PostModel(
 	if err != nil {
 		return nil, err
 	}
+	workspaceID := 1
+	if req.WorkspaceName != nil {
```

Comment:
I suspect it's going to be useful for web to support WorkspaceId in `PostModel` and `PatchModel` as well, not just `Get`.